catkin_tools/common.py

```diff
@@ -640,7 +640,7 @@ def find_enclosing_package(search_start_path=None, ws_path=None, warnings=None,
         # Find packages under the search path
         try:
             pkgs = find_packages(search_path, warnings=warnings)
-        except:
+        except RuntimeError:
```

Comment:
This is based on inspection and the doc comments on the function in question:

http://docs.ros.org/independent/api/catkin_pkg/_modules/catkin_pkg/packages.html#find_packages